dashboard/config/routes.rb

```diff
@@ -8,7 +8,7 @@
     # Routes needed for the footer on weblab share links on codeprojects
     get '/weblab/footer', to: 'projects#weblab_footer'
     get '/scripts/hosted.js', to: redirect('/weblab/footer.js')
-    get '/style.css', to: redirect('/assets/weblab/footer.css')
+    get '/style.css', to: redirect('/weblab/footer.css')
```

Comment:
TIL that the browser will follow redirects when loading css files!

Reply:
I think this redirect is something we can clean up once the page is in a good state. The route is hardcoded [here](https://github.com/code-dot-org/code-dot-org/blob/770d71e7d8c29c62d64e106ca8cc04778cf4d4f1/shared/middleware/files_api.rb#L288) and I wanted to keep the routes consistent between pegasus and dashboard so that we didn't have to make the nginx change bigger than it needed to be. Once we're ready to take codeprojects out of pegasus entirely, we can remove these redirects

Reply:
thank you Bethany, that is good to know! I'm opening a work item to track that here: https://codedotorg.atlassian.net/browse/XTEAM-389 if you happen to get to this before you move on, please feel free to close it out.